src/test/groovy/sdkman/specs/PlatformSpec.groovy

```diff
@@ -1,9 +1,13 @@
 package sdkman.specs
 
+import sdkman.stubs.UnameStub
 import sdkman.support.SdkmanEnvSpecification
 
 class PlatformSpec extends SdkmanEnvSpecification {
+	UnameStub unameStub
```

Comment:
Is it possible to rely on the base class here if we put it back into the `SdkmanEnvSpecification`? The purpose of that class is, so we don't need to create any fixture like this manually.

Reply:
The problem is that if you call `UnameStub.prepareIn` in the base class it will always create an empty `uname` file, which will break a lot of tests on non-Linux systems. Calling `prepareIn` only makes sense if you also call `forKernel` and `forMachine` *and* `build`.

Reply:
Or maybe I misunderstood you 😄 .

Reply:
The fixture is now created in the base class again.